cypress.config.ts

```diff
@@ -12,6 +12,7 @@ module.exports = defineConfig({
   responseTimeout: 60000,
   viewportWidth: 2000,
   viewportHeight: 1320,
+  retries: 1,
```

Comment:
This will affect every test config. Do we want to add this? I think someone with more experience in the codebase should comment